index.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<head>
+    <title>1SWASP J1407b</title>
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>1SWASP J1407b</h1>
+    <h3>atau dikenal Super Saturnus</h3>
+    <p>
+        Diluar angkasa yang luas nan indah, terdapat banyak sekali bebatu-batuan, gas, planet, bintang, galaksi ataupun
```

Comment:
Saran perbaikan agar sesuai dengan kaidah penulisan yang benar.
```suggestion
        Di luar angkasa yang luas nan indah, terdapat banyak sekali bebatuan, gas, planet, bintang, galaksi ataupun
```